doc/lua_api.md

```diff
@@ -5948,8 +5948,11 @@ Environment access
 * `minetest.add_entity(pos, name, [staticdata])`: Spawn Lua-defined entity at
   position.
     * Returns `ObjectRef`, or `nil` if failed
+    * Entities with `static_save = true` can be added also 
+      to unloaded/nongenerated blocks.
```

Comment:
to make sure that nobody thinks they are the same, an explicit "and" instead of the slash would be good

also, I think it should be spelled "non-generated"?